### PR TITLE
DC-1644 send reply through backend

### PR DESCRIPTION
**Pull request by sammhmrc**
Posts message through to backend

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -97,23 +97,43 @@ auditing {
...
+
+assets {
+  version = "2.149.0"
+  version = ${?ASSETS_FRONTEND_VERSION}
+  url = "http://localhost:9032/assets/"
+}
+
+contact-frontend {
+  host = "http://localhost:9250"
+}
```
[the remark this answers is not in the document]
Comment (elabeca) [reply conf/application.conf]: not sure what that is

File: app/uk/gov/hmrc/twowaymessageadviserfrontend/models/ReplyDetails.scala [hunk 2]
```diff
@@ -18,7 +18,7 @@ package models
 
 import play.api.libs.json._
 
-case class ReplyDetails (reply: String)
```
[the remark this answers is not in the document]
Comment (lxol) [reply app/uk/gov/hmrc/twowaymessageadviserfrontend/models/ReplyDetails.scala]: bug, yes it should be content

File: app/uk/gov/hmrc/twowaymessageadviserfrontend/controllers/ReplyController.scala [hunk 3]
```diff
@@ -64,8 +66,10 @@ class ReplyController @Inject()(appConfig: FrontendAppConfig,
         (replyDetails) => {
 
           Logger.debug(s"replyDetails: ${replyDetails}")
+          twoWayMessageConnector.postMessage(replyDetails, id.stringify).map {
```
Comment (lxol) [inline app/uk/gov/hmrc/twowaymessageadviserfrontend/controllers/ReplyController.scala]: we should probably handle failures here and add some tests

Comment (sammhmrc) [reply app/uk/gov/hmrc/twowaymessageadviserfrontend/controllers/ReplyController.scala]: Should bubble up and be caught by the `recoverWith` block below. IRT tests, we'd like to take out some tech debt and push those in later. Getting this in faster would be ideal

**Pull request merged**